diff --git a/lvgl/src/drivers/display/drm/lv_linux_drm.c b/lvgl/src/drivers/display/drm/lv_linux_drm.c
index 29e0168d9..c488b71bf 100644
--- a/lvgl/src/drivers/display/drm/lv_linux_drm.c
+++ b/lvgl/src/drivers/display/drm/lv_linux_drm.c
@@ -118,7 +118,7 @@ lv_display_t * lv_linux_drm_create(void)
     LV_ASSERT_MALLOC(drm_dev);
     if(drm_dev == NULL) return NULL;

-    lv_display_t * disp = lv_display_create(800, 480);
+    lv_display_t * disp = lv_display_create(1024, 600);
     if(disp == NULL) {
         lv_free(drm_dev);
         return NULL;
