diff --git a/src/drivers/display/drm/lv_linux_drm.c b/src/drivers/display/drm/lv_linux_drm.c
index 29e0168d9..752c291a5 100644
--- a/src/drivers/display/drm/lv_linux_drm.c
+++ b/src/drivers/display/drm/lv_linux_drm.c
@@ -163,7 +163,7 @@ void lv_linux_drm_set_file(lv_display_t * disp, const char * file, int64_t conne
      */
     lv_display_set_resolution(disp, hor_res, ver_res);
     lv_display_set_buffers(disp, drm_dev->drm_bufs[1].map, drm_dev->drm_bufs[0].map, buf_size,
-                           LV_DISPLAY_RENDER_MODE_DIRECT);
+                           LV_DISPLAY_RENDER_MODE_PARTIAL);
 
     if(width) {
         lv_display_set_dpi(disp, DIV_ROUND_UP(hor_res * 25400, width * 1000));
@@ -371,6 +371,7 @@ static int drm_dmabuf_set_plane(drm_dev_t * drm_dev, drm_buffer_t * buf)
 
         drm_add_crtc_property(drm_dev, "MODE_ID", drm_dev->blob_id);
         drm_add_crtc_property(drm_dev, "ACTIVE", 1);
+        drm_add_crtc_property(drm_dev, "ROTATION", 2);]
 
         flags |= DRM_MODE_ATOMIC_ALLOW_MODESET;
 
